options/locale/locale_en-US.ini

```diff
@@ -344,6 +345,8 @@ org_still_own_repo = "This organization still owns one or more repositories; del
 
 target_branch_not_exist = Target branch does not exist.
 
+SSPISeparatorReplacement = Separator
```

Comment:
Wrong casing and also  does not seems to be used anywhere

Reply:
Its only used if admin enters a separator, that contains characters not allowed in usernames:
![SSPISeparatorReplacementError](https://user-images.githubusercontent.com/7578087/67747962-8fd63600-fa32-11e9-946d-15be7b0f6b74.png)

Turns out casing of field names can be changed with the `locale:` tag inside the struct.

Should be fixed by commit [Add locale tag for the SSPISeparatorReplacement field with proper casing](https://github.com/quasoft/gitea/commit/e1c2b7b6d3896abb0246f0c5e8fdcc9ad486ebb5)

Reply:
Oh, I'm very sorry that is my fault... Naming was correct just placement was not rights. It should be moved above where all field names are: https://github.com/go-gitea/gitea/pull/8463/files#diff-f65c5e89d3eccc273f66ad63e7852b64L300

Reply:
Can you please revert and just move it before error validation messages?

Reply:
Reverted and moved next to other fields with [Revert casing of SSPISeparatorReplacement field in locale file, moving it up, next to other form fields](https://github.com/go-gitea/gitea/pull/8463/commits/cd390c66bea798c8522672a6f91a97b9d228690e).

Is that the correct place for field names?

Reply:
Yes, looks right now